package.json

```diff
@@ -15,11 +15,13 @@
     "@11ty/eleventy-navigation": "^0.3.3",
     "@tailwindcss/typography": "^0.5.2",
     "autoprefixer": "^10.4.7",
+    "daisyui": "^2.15.3",
     "postcss": "^8.4.14",
     "tailwindcss": "^3.1.2",
-    "vite": "^2.9.12",
-    "daisyui": "^2.15.3"
+    "vite": "^2.9.12"
```

Comment:
hey Fershad - what does vite do here?

I get that it is a bundler that uses esbuild under the hood, but I can't tell what it's doing in this PR, and I can't see it being explicitly called in any npm commands or config files.

If it's not actively being used, could we remove, and add it later with a bit of explanation about how to use it in development?

Reply:
I think it came along for the ride when copying the DaisyUI getting started code for 11ty. I've removed it and don't see any ill effects.